### PR TITLE
Smash Magazine Heatmap Project

### PR DESCRIPTION
**Pull request by nusretbutunay**
This project consists of building a heatmap of the Smashing magazine website

In this project:

  1. Float, flex and grid properties used to align elements on the page
  2. Variations of gray colors used
  3. Box Show property used to give contrast look on the page
  4. Transform property used to rotate the images on the page

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @nusretbutunay and @Maryjanee, great job with this project :+1: Descriptive commit messages :slightly_smiling_face: Linters and validator passed successfully :100: Let us get our project approved by addressing the comments below:

- Kindly replace all divs with the class ```article``` with the ```<article>``` so we can improve the use of HTML semantics
article
- Kindly fix the horizontal scroll. A quick fix is to set the main width to 100% so it can fit on all screen
hroi
- The rotated image has overlapped with some of the text. Kindly check image below
pos
- The PR and the README description can be improved, kindly edit the PR so we have a descriptive PR. We can check the project requirements for a hint. Example; In this project we did: 1. Positioned elements with float, grid, and flex, etc 
- Kindly fix the author section where we have your social media handles. I could be indentation problem, let us fix it and more our README file more professional

Kindly make these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: :woman_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @nusretbutunay  and @Maryjanee ! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There is just one issue that needs to be resolved before you can move on: ⬇️ 
****

- [x] As was requested by the previous TSE, the rotated image is still overlapping with the "community links" text. Please fix this.

Screen Shot 2020-06-21 at 11 41 28 AM
****
### [OPTIONAL]

- While your linters are showing no errors, they are showing some hints. Feel free to use the "Webhint Report" in the linters checks to apply those hints.
Screen Shot 2020-06-21 at 11 43 43 AM

_P.S. all requested changes marked by [OPTIONAL] are not required. It is up to you whether or not you would like to do them_
****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## Hello @nusretbutunay and @Maryjanee 🖐. You have done an incredible job making this project 🙌👏✨!

## It's a 👍 from me. 

### Before you merge, I would request you consider making these optional changes.

#### [OPTIONAL]

- In `bottom-article-container`, put a border around each of the `article` elements to make it obvious that they are separate.
- In `section.click`, change the color of the texts. It's hard to see right now.
- Increase the height of the footer to match the original.
- There are some invalid CSS property values. Check the comments below.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+  --black: #000;
+  --secondary-text-color: grey;
+}
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: 0;
```
Comment (tirthajyoti-ghosh) [inline style.css]: `box-sizing` doesn't accept `0` as its value.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+  margin-bottom: 1.5rem;
+  line-height: 2.5rem;
+}
+
+.flex-container {
+  display: flex;
+}
+
+.text-align-left {
+  text-align: left !important;
```
Comment (tirthajyoti-ghosh) [inline style.css:51]: Remove `!important`. It's a bad practice.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+
+.call-wrapper {
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.call-wrapper h1 {
+  background-color: var(--main-text-color);
+  padding: 1rem;
+  box-shadow: 0 200px 171 -92px rgba(0, 0, 0, 0.75);
```
Comment (tirthajyoti-ghosh) [inline style.css:201]: Remember to add units. Unit missing for `171`.

**Review by tirthajyoti-ghosh — APPROVED** [review]
Consider applying these optional changes before merging.

Onto the next project 🚀.

Happy coding 👨‍💻

**Pull request merged**